sdk/webpubsub/Microsoft.Azure.Functions.Worker.Extensions.WebPubSub/src/Models/Output/WebPubSubAction.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Abstract class of operation to invoke service.
+    /// </summary>
+    public abstract class WebPubSubAction
```

Comment:
Do we not need all of these factory methods? https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/api/Microsoft.Azure.WebJobs.Extensions.WebPubSub.netstandard2.0.cs#L112

Reply:
I think the common practice is not to mix use in-process and isolated-process extension packages. So either customers have to declare themselves or we do this for convenience.

See sample without built-in types and customer have to do in this way in early SignalR extension: https://github.com/Azure/azure-functions-dotnet-worker/blob/main/samples/Extensions/SignalR/SignalRFunction.cs